glean-core/src/lib.rs

```diff
@@ -36,10 +36,9 @@ mod internal_pings;
 pub mod metrics;
 pub mod ping;
 pub mod storage;
-mod util;
-
 #[cfg(feature = "upload")]
-mod upload;
+pub mod upload;
```

Comment:
What's the reason to make this `pub`?
I don't think we need to, as we expose for example `get_upload_task` as a method on `Glean`.

Reply:
Hm, I am not sure really. If I don't make this public tho, android tests fail.

Reply:
huh? it shouldn't make _tests_ fail.

Reply:
Agree.

Reply:
I will remove this, it won't break tests now because of the feature flag. I can understand what is going on when I start working on the android bindings. What do you think?

Reply:
Sure! 
Once we build the platform layer for this we need to think how we are going to deal with this feature flag anyway.